README.md

```diff
@@ -15,7 +15,7 @@ In short term we are working on two main goals of the project:
 
 This is a LINCC Frameworks project - find more information about LINCC Frameworks [here](https://www.lsstcorporation.org/lincc/frameworks)
 
-To learn about the usage of the package, consult the [Documentation](https://lsstseries.readthedocs.io/en/latest/index.html).
+To learn about the usage of the package, consult the [Documentation](https://tape.readthedocs.io/en/latest/index.html).
```

Comment:
Have you already reserved this? I figured if tape.readthedocs was taken we'd need to do something like LINCC-tape.readthedocs for the webpage names.

Reply:
Not reserved, but looks free. We'll see if rtd runs any issues with the first build from main